centreon/packages/ui/src/Graph/LineChart/index.tsx

```diff
@@ -25,12 +25,16 @@ dayjs.extend(timezonePlugin);
 interface Props extends Partial<LineChartProps> {
   curve?: CurveType;
   data?: LineChartData;
+  disabledThresholds?: boolean;
```

Comment:
Can we handle this prop the other way around? it'd be more logic to reason about in my opinion: e.g displayThresholds or thresholdsEnabled